### PR TITLE
[feature] Replace nlogs with brief description text [OSF-8866]

### PR DESCRIPTION
**Pull request by alexschiller**
## Purpose
Nlogs aren't useful. Replace them with something more awesomer.

## Changes
1. Remove nlogs and nlogs annotation (hey that's probably a [minor] speed increase too).
2. Add description text that truncates at 150 characters (to be adjusted in demo)
BEFORE:
screen shot 2017-11-13 at 2 07 57 pm
AFTER (No description):
screen shot 2017-11-13 at 2 01 07 pm

AFTER (<150 chars):
screen shot 2017-11-13 at 2 01 55 pm

AFTER (>150 chars):
screen shot 2017-11-13 at 2 00 15 pm

## Side effects
The log count was hidden for anonymous views. I don't know why. I made this visible.

## Ticket

https://openscience.atlassian.net/browse/OSF-8866

## QA Notes
1. Check empty descriptions
2. Check > X characters (where X is the final cutoff value)
3. Check < X characters

Confirm that the description should be shown for anonymous VOLS. I can't see any reason it shouldn't be shown.

### CODE REVIEW

File: website/project/views/node.py [hunk 1]
```diff
@@ -795,12 +795,12 @@ def _view_project(node, auth, primary=False,
     if embed_registrations:
         data['node']['registrations'] = [
             serialize_node_summary(node=each, auth=auth, show_path=False)
-            for each in node.registrations_all.order_by('-registered_date').exclude(is_deleted=True).annotate(nlogs=Count('logs'))
```
Comment (sloria) [inline website/project/views/node.py]: Drive-by: The `nlogs` key should also be removed from serialization in `serialize_node_summary`. If you remove this annotation but don't remove the serialization, we'll be doing an expensive log count query for each node.

Comment (sloria) [reply website/project/views/node.py]: You did this. Disregard.

**Comment by brianjgeiger** [comment]
@alexschiller Conflicts and errors

File: website/views.py [hunk 2]
```diff
@@ -114,10 +114,9 @@ def serialize_node_summary(node, auth, primary=True, show_path=False):
             'parent_title': parent_node.title if parent_node else None,
             'parent_is_public': parent_node.is_public if parent_node else False,
             'show_path': show_path,
-            # Read nlogs annotation if possible
-            'nlogs': node.nlogs if hasattr(node, 'nlogs') else node.logs.count(),
             'contributors': contributor_data['contributors'],
             'others_count': contributor_data['others_count'],
+            'description': node.description if len(node.description) < 150 else node.description[0:150] + '...',
```
Comment (brianjgeiger) [inline website/views.py]: I presume you've verified the edge cases on this, yes?

Comment (alexschiller) [reply website/views.py]: IIRC, yes, but it has been a month, so...
